### PR TITLE
refactor: Fix clippy's len_zero errors

### PR DESCRIPTION
**Pull request by jbajic**
Addressing issue #8631

### CODE REVIEW

**Review by pugachAG — APPROVED** [review]
LGTM, but please fix formatting as reported by [buildkite](https://buildkite.com/nearprotocol/nearcore/builds/27260) sanity checks. You can use either `cargo fmt` or `./scripts/formatting` to help you.
You can also link #8631 in Development section, so it will be closed when this PR is merged.
In order to get the PR merged you have to add `S-automerge`. Then it will be automatically merged once you have approval from one of the code owners and buildkite is green.

**Review by nagisa — APPROVED** [review]

**Comment by Longarithm** [comment]
9000 GET!

**Pull request merged**
